frontend/kick-covid-19/src/App.js

```diff
@@ -14,6 +15,9 @@ const App = () => (
       <Route exact path="/">
         <Home />
       </Route>
+      <Route path="/posts/:id">
+        <Posts />
+      </Route>
       <Route exact path="/blog">
         <Blog />
```

Comment:
I would rename this to `PostList` or `QuestionList` and then rename `Posts` to `Post` or `Question` or  `QuestionDetail` or something similar since this isn't really a blog.